packages/api/src/trpc.ts

```diff
@@ -141,3 +174,22 @@ export const protectedProcedure = t.procedure
       },
     });
   });
+
+export const adminProcedure = t.procedure
```

Comment:
You could probably refactor this by extending the authenticated procedure, but this is fine for now.